data/fro/Zink_1984_Cleriadus/stemma.gv

```diff
@@ -0,0 +1,34 @@
+digraph {
+
+O -> x; 
+O -> y;
+
+y -> P3;
+y -> L;
+
+y -> T [style="dashed"];
+x -> T [style="dashed"];
+
+x -> A; 
+x -> V; 
+x -> xprime;
+xprime [label="x'"]
+
+xprime -> P1;
+xprime -> P2;
+
+V -> xprimeprime;
+xprimeprime [label="x''"]
+
+xprimeprime -> B; 
+xprimeprime -> C;
+
+B -> Vér [style="dashed"];
```

Comment:
Here, and for T, I wonder if there is a primary, more important filiation, or if both should be considered in the same way ? In other words, should we have two dashed lines, one full and one dash, or two full ?

Reply:
We should draw two full lines from x'' to B and C (as for x, to A, V and x'; or for y, to P3 and L etc.) and two dashed lines from B to Vér, and from C to Vér; I cite the editor: "...c'est sur B et C [...] que s'est fondée l'éd. Vérard pour établir la première partie du roman. Les traits partagés abondent [...]. Mais les imprimés présentent aussi [...] des traits qu'on trouve seulement dans B ou seulement dans C [...]. Il s'agit donc d'une composition hybride."